hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HddsServerUtil.java

```diff
@@ -583,4 +584,29 @@ public static IOException toIOException(String msg, RocksDBException e) {
   public static void addSuppressedLoggingExceptions(RPC.Server server) {
     server.addSuppressedLoggingExceptions(ServerNotLeaderException.class);
   }
+
+  /**
+   * Build javaProcessBuilder.
+   * @param jvmArgs
+   * @param classpath
+   * @param className
+   * @param args
+   * @return ProcessBuilder
+   */
+  public static ProcessBuilder getJavaProcess(List<String> jvmArgs,
+                                           String classpath,
+                                           String className,
+                                           List<String> args) {
+    String javaHome = System.getProperty("java.home");
+    String javaBin = String.format("%1$s%2$sbin%2$sjava",
+            javaHome, File.separator);
```

Comment:
fancy stuff.

Add a comment saying we are trying to guess the `java` executable path here?

Reply:
Removed function not needed